### ISSUE TITLE
Saving an Edit gives a 500: error dialog, but edit is saved successfully

### ISSUE THREAD

**Issue by sgornick** [gab-ai-inc/gab-dissenter-extension#20]
On Chromium, with Dissenter v0.1.1 extension, if I try to edit a Dissent comment of mine, I get an error dialog that shows:
```
An embedded page at dissenter.com says
500:   
   [OK]
```
The edit made is successful (i.e., the change I made shows up in edit history), so this appears to be an undesired artifact.

The message was entered with the "Share to Gab" option enabled, which is a new feature in v0.1.1, so not sure if that is relevant to this issue.

**Comment by mgabdev** [comment]
Hi @sgornick can you please confirm whether this is still an issue or not. Thanks.

**Comment by dotnetspec** [comment]
Same problem except it **isn't** saving the change.

![image](https://user-images.githubusercontent.com/27340988/55445251-0b7f1a00-55ed-11e9-8a7c-7d90c7f3b743.png)

Linux Mint 18.3 with Chrome
Version 73.0.3683.86 (Official Build) (64-bit)

**Comment by sgornick** [comment]
> Hi @sgornick can you please confirm whether this is still an issue or not. I am not seeing anymore in the latest version. Thanks.

The error message of 500 appears to have been fixed, even though I made no changes on my end.  So I presume it was a back-end issue.

And the edit does get saved for me.  i.e., If I look at my post on Dissenter.com/user/[me]  I can see the comment, including my edit.

So I think this issue can be marked resolved.     What is happening though is the edit is not propagating to the comment on gab.com/@[me] as well.  I think I will open a new issue on that.
